.travis.yml

```diff
@@ -10,7 +10,7 @@ addons:
   postgresql: '9.5'
 env:
   global:
-  - DATABASE_URL="postgresql://postgres:@localhost:5432/service_portal_test?encoding=utf8&pool=5&wait_timeout=5"
+  - DATABASE_URL="postgresql://postgres:@localhost:5432/service_portal_api_test?encoding=utf8&pool=5&wait_timeout=5"
```

Comment:
@Fryguy - Was this the issue that wasn't allowing the deploy to finish?

Reply:
No...it was actually the ENV vars in Travis that were still set to the old deployment cluster